docs/streams/quickstart.html

```diff
@@ -57,7 +57,7 @@ <h1>Run Kafka Streams Demo Application</h1>
 // represent lines of text (for the sake of this example, we ignore whatever may be stored
 // in the message keys).
 KStream&lt;String, String&gt; textLines = builder.stream("streams-plaintext-input",
-    Consumed.with(stringSerde, stringSerde);
```

Comment:
nit: maybe we can reformat this for readability like this:
```
KStream&lt;String, String&gt; textLines = builder.stream(
  "streams-plaintext-input",
  Consumed.with(stringSerde, stringSerde)
);
```

Reply:
Thanks @vvcephei. I'll update later.